docs/console-api/daily-reports-depricated.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Daily Reports Depricated
```

Comment:
will need to redo file names and everything, but its spelled "deprecated"

Reply:
Lol I fixed the spelling but didn't push